src/js/stores/panel.js

```diff
@@ -248,6 +248,7 @@ define(function (require, exports, module) {
             var centerOffsets = this.getCenterOffsets(),
                 windowWidth = window.document.body.clientWidth,
                 windowHeight = window.document.body.clientHeight;
+
```

Comment:
I think that's because the new line has spaces, which is not considered as extra line break.